wikitemplate-ios.md

```diff
@@ -1,119 +1,122 @@
 
 ## Sync
-1. [ ] Ensure you are able to scan the QR code and sync with laptop
-2. [ ] Ensure the bookmarks from laptop shows up on the mobile after sync completes
-3. [ ] Add a bookmark on mobile and check if it gets synced to the laptop
+- [ ] Ensure you are able to scan the QR code and sync with laptop
+- [ ] Ensure the bookmarks from laptop shows up on the mobile after sync completes
+- [ ] Add a bookmark on mobile and check if it gets synced to the laptop
 
 ## Bravery settings
 
-1. [ ] Check that HTTPS Everywhere works by loading https://https-everywhere.badssl.com/
-2. [ ] Turning HTTPS Everywhere off and shields off both disable the redirect to https://https-everywhere.badssl.com/
-3. [ ] Check that block ad and unblock ad works on http://slashdot.org
-4. [ ] Check that toggling to blocking and allow ads works as expected.
-5. [ ] Test that clicking through a cert error in https://badssl.com/ works.
-6. [ ] Test that Safe Browsing works (http://downloadme.org/)
-7. [ ] Turning Safe Browsing off and shields off both disable safe browsing for http://downloadme.org/.
-8. [ ] Enable block script globally from settings, Visit https://brianbondy.com/, nothing should load. Tap on Shields and disable block script, page should load properly
-9. [ ] Test that preferences default Bravery settings take effect on pages with no site settings.
-10. [ ] Test that turning on fingerprinting protection in preferences shows 1 fingerprints blocked at https://browserleaks.com/canvas . Test that turning it off in the Bravery menu shows 0 fingerprints blocked.
-11. [ ] Test that 3rd party storage results are blank at https://jsfiddle.net/7ke9r14a/7/ when 3rd party cookies are blocked.
-12. [ ] Test that audio fingerprint is blocked at https://audiofingerprint.openwpm.com/ when fingerprinting protection is on.
+- [ ] Check that HTTPS Everywhere works by loading https://https-everywhere.badssl.com/
+- [ ] Turning HTTPS Everywhere off and shields off both disable the redirect to https://https-everywhere.badssl.com/
+- [ ] Check that block ad and unblock ad works on http://slashdot.org
+- [ ] Check that toggling to blocking and allow ads works as expected.
+- [ ] Test that clicking through a cert error in https://badssl.com/ works.
+- [ ] Test that Safe Browsing works (http://downloadme.org/)
+- [ ] Turning Safe Browsing off and shields off both disable safe browsing for http://downloadme.org/.
+- [ ] Enable block script globally from settings, Visit https://brianbondy.com/, nothing should load. Tap on Shields and disable block script, page should load properly
+- [ ] Test that preferences default Bravery settings take effect on pages with no site settings.
+- [ ] Test that 3rd party storage results are blank at https://jsfiddle.net/7ke9r14a/7/ when 3rd party cookies are blocked.
+
+**Fingerprinting Tests (Enable fingerprinting protection in Settings)**
```

Comment:
Should the mobile templates use the same new FP cases as `wikitemplate.md`?

Example: `got canvas fingerprint 0`  and  `got webgl fingerprint 00`

Reply:
For iOS there is only Canvas check done so the test url is also different compared to laptop. Can add it back later once security team says its the same result for mobile as well.

Reply:
👍

Reply:
the changes that @srirambv made to the wiki template recently should probably be reverted since the code change to fingerprinting protection in browser-laptop (https://github.com/brave/browser-laptop/pull/11784) was reverted